tutorials/nlp/Dialogue.ipynb

```diff
@@ -0,0 +1,5269 @@
+        "\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "FJk_UAyeEq3B"
+      },
+      "source": [
+        "\n",
+        "## Download the dataset and convert it to the NeMo format\n",
+        "\n",
+        "In this tutorial we are going to use a virtual assistant interaction data set that can be downloaded from here: https://github.com/xliuhw/NLU-Evaluation-Data.\n",
+        "There are about 10K training and 1K testing queries which cover 64 various Intents and 55 Slots. \n",
+        "\n",
+        "To work with NeMo NLP classification model, this dataset should be first converted to the NeMo format, which requires next files:\n",
+        "- **dict.intents.csv** - list of all intent names in the data. One line per an intent name.\n",
+        "- **dict.slots.csv** - list of all slot names in the data. One line per a slot name. It is possible to use both: B- I- notations, for separating between first and intermediate tokens for multi token slots. Or just use one slot type for each token of multi token slot. Our recommendation is to use later one, since it is simpler and there is no visible degradation in performance.\n",
+        "- **train.tsv/test.tsv** - contain original queries, one per line, and intent number separated by tab. For example: `what alarms do i have set right now\t0`. Intent numbers are according to the intent line in the intent dictionary file (dict.intents.csv) starting from 0. First line of these files contains a header line: `sentence \\tab label`.\n",
+        "- **train_slot.tvs/test_slot.tsv** - contain one line per a query, where instead each token there is a number of the token from the slots dictionary file (dict.slots.csv), starting from 0. Last 'out-of scope' token is usually located in the last line of the dictionary. Example: `54 0 0 54 54 12 12` (numbers separated by space). No header line in these files.\n",
+        "\n",
+        "NeMo provides **import_dataset.py** converter for few reference datasets (Assistant / Atis / Snips) which converts them to the NeMo data format for the Intent and Slot classification model. If you have your own annotated dataset in a different format, you will need to write a data converter. Possible recommended format for your own annotation, is to have one text file per all examples of one intent. With one line per query in a form like: `did i set an alarm to [alarm_type : wake up] in the [timeofday : morning]`, using brackets to define slot names. This is very similar to the assistant format from this example and you can use its converter to NeMo format with small changes. \n",
+        "\n",
+        "You can run this utility as follows:\n",
+        "\n",
+        "**python examples/nlp/intent_slot_classification/data/import_datasets.py --dataset_name=assistant --source_data_dir=source_dir_name --target_data_dir=target_dir_name**"
+      ]
+    },
+    {
+      "cell_type": "code",
```

Comment:
Clear the cell outputs from the notebook

Reply:
Sure